testgres/api.py

```diff
@@ -28,7 +28,7 @@
 ...             replica.catchup()  # wait until changes are visible
 ...             print(replica.execute('postgres', 'select count(*) from test'))
 PostgresNode(name='...', port=..., base_dir='...')
-[(3,)]
+[(3L,)]
```

Comment:
is this necessary?

Reply:
I think it was when I tested with doctest. But now when I ran doctest it showed `[(3,)]` so I am confused :) Will change it back then as it shouldn't be part of this pull request anyway.